src/modules/85-iac/utils/index.ts

```diff
@@ -0,0 +1,3 @@
+import { getStyles } from './getStyles'
+
+export { getStyles }
```

Comment:
This file seems a tad redundant. Why not just import the `getStyles` function direct from the `getStyles` when you need it?

You could go even more terse if you fancy it
```suggestion
export { getStyles } from './getStyles'
```